docs/usage/tutorials/CSKD.rst

```diff
@@ -0,0 +1,33 @@
+======================
+Regularizing Class-wise Predictions via Self-knowledge Distillation using KD_Lib
+======================
+
+To implement the most basic version of knowledge distillation from `Regularizing Class-wise Predictions via Self-knowledge Distillation <https://arxiv.org/abs/2003.13964>`_
+and plot losses
```

Comment:
Maybe add some explanation, about the paper or just paste the schematic of the whole paper summarizing it

Reply:
Sure. I'll add the summary and key contributions of the paper.